ffmpeg/extras.c

```diff
@@ -176,6 +176,10 @@ int lpms_get_codec_info(char *fname, pcodec_info out)
       }
       out->width  = ic->streams[vstream]->codecpar->width;
       out->height = ic->streams[vstream]->codecpar->height;
+      
+      AVRational frame_rate = av_guess_frame_rate(ic,ic->streams[vstream],NULL);
```

Comment:
Just to make sure, did you check the performance of this call? Won't it slow down the input file analysis?

Reply:
I have updated to use `avg_frame_rate` in the `AVStream` from the `avformat_find_stream_info` call already being done.  Duration is in the `AVFormatContext` as well so don't need to calculate.  Want to note that the `nb_frames` was not populated on webm files  I think this is a container spec driven difference compared to mp4 files.